equipment_handler/source/tempsense_athena.c

```diff
@@ -23,25 +23,23 @@
 #define CHANNEL_LOCAL 0
 #define CHANNEL_REMOTE 1
 
-uint8_t tmp_addr[6] = {0x1D, 0x1E, 0x2A, 0x4C, 0x4F, 0x4E};
+uint8_t tmp_addr[6] = {0x49, 0x4A};
 
-void inittemp_all(void) {
+void inittemp_all(void){
     int i;
     int delay;
-    for (i = 0; i < 6; i++) {
+    for(i=0;i<2;i++){
         tmp421_init_client(tmp_addr[i]);
-        for (delay = 0; delay < 0x1000; delay++)
-            ; // temporary fix... don't want delay down the road
+        for (delay = 0; delay < 0x1000; delay++);//temporary fix... don't want delay down the road
```

Comment:
Since we've started only running code like this in task contexts, you should add a vTaskDelay